custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -346,7 +346,8 @@ def _prepare_prediction_server_or_batch_pipeline(self, run_language):
         return functional_pipeline_str
 
     def _prepare_fit_pipeline(self, run_language):
-        if not self.options.negative_class_label:
+
+        if not self.options.negative_class_label and not self.options.unsupervised:
```

Comment:
does using class labels fail for unsupervised?

Reply:
possibly_intuit_order will throw an error if there's no target- that's why this check is here

Reply:
what if you made it not throw an error though.

Reply:
I can probably do that